things/database.py

```diff
@@ -188,9 +188,10 @@ def get_tasks(  # pylint: disable=R0914
         start = start and start.title()
 
         # Validation
-        validate("type", type, [None] + list(TYPE_TO_FILTER))  # type: ignore
-        validate("status", status, [None] + list(STATUS_TO_FILTER))  # type: ignore
         validate("start", start, [None] + list(START_TO_FILTER))  # type: ignore
+        validate("status", status, [None] + list(STATUS_TO_FILTER))  # type: ignore
```

Comment:
do we need "type: ignore" on each of these lines?

Reply:
Pyright isn't very happy:
```
Operator "+" not supported for types "list[None]" and "list[str]" (reportGeneralTypeIssues)
```
We could also disable it generally in `setup.cfg`, however, type checking itself is not a bad thing...

Reply:
@AlexanderWillner Thanks for the clarification. That sounds like a bug in Pyright to me.

Reply:
Seems it's intended to work like this: https://github.com/microsoft/pyright/issues/1073

Reply:
Thanks for the link! I don't get it though, unlike that example, we haven't written any types anywhere as far as I can see. How would they infer that?